corehq/apps/email/migrations/0003_emailsettings_password_cbc.py

```diff
@@ -12,8 +12,9 @@ def copy_and_reencrypt_password_to_password_cbc(apps, schema_editor):
     EmailSettings = apps.get_model('email', 'EmailSettings')
 
     for email_settings in EmailSettings.objects.all():
-        email_settings.password = reencrypt_ecb_to_cbc_mode(email_settings.password, f'${ALGO_AES}$')
-        email_settings.save()
+        if not email_settings.password.startswith(f'${ALGO_AES_CBC}$'):
```

Comment:
These records could be filtered out in the database query.

Reply:
That's better [1601646](https://github.com/dimagi/commcare-hq/pull/35403/commits/16016467bd51ac782eb2ce363e08b5b0b4ae0de1)